apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,24 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "mbed.h"
+
+// Declare global variables
+CANBus can(CAN1_RX, CAN1_TX);
+CANMsg msg;
+TutorialServo servo(PA_1);
+uint8_t convertedMessage = 0;
+
+// main() runs in its own thread in the OS
+int main() {
+  while (true) {
+    if (can.read(msg)) {
+      msg >> convertedMessage;
```

Comment:
Better to use CANMsg's `getPayload()` function (eg. `msg.getPayload(convertedMessage);`). The data type should be float. See the arm app for an example

Reply:
*float instead of uint8_t